### PR TITLE
Fixes #3513

### LINKED ISSUE #3513: Incompatible add-ons inside a collection have layout issues
Steps to reproduce:
1. Load details page for a collection that contains incompatible add-ons i.e. https://addons-dev.allizom.org/en-US/firefox/collections/mozilla/privacy/?page=2
2. Observe the incompatible add-on

Expected results:
There are no display or layout issues.

Actual results:
There is a layout issue in what concerns the availability/incompatibility message.

Notes/Issues:
Also the "View other versions" is not blue as in add-on details page.
Verified on FF48.0.2(Win 7). Reproducing on all add-ons servers.
Screenshots for this issue: 
coll1
coll2

#3437

### PR DESCRIPTION
**Pull request by tsl143**
Fixes #3513

### CODE REVIEW

File: static/css/restyle/restyle.less [hunk 1]
```diff
@@ -1774,6 +1773,11 @@ h3.author .transfer-ownership {
   .addon.hovercard & {
     border: 1px solid #ebebeb;
   }
+
+  .more-versions {
+    color: #0996f8;
+    margin: 0 .25em 0 0.20em;
```
Comment (tofumatt) [inline static/css/restyle/restyle.less]: This is a nitpick but could you be consistent with the format here? I'm not sure if the file overall prefers or omits leading zeros, but please use/don't use them in both cases to be consistent :-)

Also, we seem to use `px` in most places, why use `em` here?

Comment (tsl143) [inline static/css/restyle/restyle.less]: Even I was a bit in dilemma to use that, actually I just referred from `.button span` CSS rule `0 .25em 0 -.25em`
I will put the same in px now

**Comment by tsl143** [comment]
@tofumatt review?

**Comment by tsl143** [comment]
Before
before
After
after

@tofumatt I needed to add `!important` in `.more-versions a`  as `.disabled a[href]` already has `color: #888!important;` rule.

**Comment by tsl143** [comment]
@tofumatt already updated :)

**Pull request merged**